helm-chart-sources/pulsar/templates/utils/health-configmap.yaml

```diff
@@ -31,6 +31,12 @@ data:
     #!/bin/bash
     {{- if .Values.enableTokenAuth }}
     curl -s --max-time {{ .Values.proxy.probe.timeout }} --fail -H "Authorization: Bearer $(cat /pulsar/token-superuser/superuser.jwt | tr -d '\r')" http://localhost:8080/metrics/ > /dev/null
+    {{- else if .Values.openid.enabled }}
+    # Passing credentials in format client_id:client_secret using basic auth:
+    TOKEN="{{ .Values.openid.client_id }}":{{ .Values.openid.client_secret }}" | base64
+    # NOTE: It's probably insecure to store these credentials in here. What's the better way to do this?
+    #       We could copy the client_id and client_secret to separate Secret object entries and pull them that way, perhaps.
+    curl -s --max-time {{ .Values.proxy.probe.timeout }} --fail -H "Authorization: Basic $TOKEN" http://localhost:8080/metrics/ > /dev/null
```

Comment:
As we discussed offline, this approach won't work. I think we'll want to integrate long lived token auth and oauth2 for now.

Reply:
haha sorry, I thought I removed this change. I'm glad you caught it.